### ISSUE TITLE
IOS 'show cdp neighbor detail' parser fails when double-quote is in platform name

### ISSUE THREAD

**Issue by jamiecaesar** [CiscoTestAutomation/genieparser#374]
Some devices use `"` in the platform name, which causes the IOS parser for `show cdp neighbor detail` to throw `SchemaMissingKeyError`.   An example of the output is:
```
Device ID: SEP009E1E43AC48
Entry address(es): 
  IP address: 10.110.13.123
Platform: "CTS-CODEC-SX80",  Capabilities: Host Phone 
Interface: GigabitEthernet3/0/38,  Port ID (outgoing port): gbe0
Holdtime : 166 sec

Version :
"ce9.1.3 75ff735 2017-07-10"

advertisement version: 2
```

Adding `"` to the `platform` capture group that is a part of the regex under the `platf_cap_re` (line 213 of show_cdp.py) variable should resolve this problem.

**Comment by jamiecaesar** [comment]
@xiaoxinz-cisco I see you are assigned to this.   I was considering trying to figure out how to work with this repository and submit a PR to fix this parser.   I wanted to check with you, though, so we don't duplicate work.   Are you OK with me taking a shot at fixing this?

**Comment by jamesditrapani** [comment]
@jamiecaesar I had already fixed this & had the branch ready to PR. However, for the sake of learning I'm happy to trash that and let you go ahead :)

**Comment by jamiecaesar** [comment]
@jamesditrapani If you already have it all written up, I'd say just go ahead and do it.  I'm still fumbling around with figuring out how all the unit tests work (I was planning on going the TDD route -- write a new test that fails, fix it, make sure all tests run, then submit), and I can fumble through that regardless of if this is fixed or not.  I should be able to review your PR to see if what you did are the same things I had in mind.

So, in short, go ahead and push your work.  The PR wasn't the part I needed to learn, so I can still take advantage of this :)

**Comment by jamiecaesar** [comment]
@jamesditrapani I didn't see any pull requests from you yet, and I think I have everything figured out, so I went ahead and submitted one just now.

**Comment by jamesditrapani** [comment]
> @jamesditrapani I didn't see any pull requests from you yet, and I think I have everything figured out, so I went ahead and submitted one just now.

All good! 👍

**Comment by jamiecaesar** [comment]
This should be fixed with the merge of PR376.
